src/vault/interfaces/secret.interface.ts

```diff
@@ -0,0 +1,41 @@
+export interface SecretContext {
+  [key: string]: any;
+}
+
+export interface SecretDigest {
+  id: string;
+  name: string;
+  updatedAt: Date;
+}
+
+export interface SecretMetadata {
+  context: SecretContext;
+  environmentId: string;
+  id: string;
+  keyId: string;
+  updatedAt: Date;
```

Comment:
This is missing the `updatedBy` object containing an id and name.

That will also need to get added to `ReadSecretMetadataResponse`